### ISSUE TITLE
Can't get resource working with Harbor v1.8.2

### ISSUE THREAD

**Issue by funkypenguin** [cathive/concourse-chartmuseum-resource#8]
Hi,

Hopefully I'm just configuring something wrong. My pipeline has an error on the chartmuseum resource, reading `unexpected end of JSON input`. When I try to run the pipeline, the **put** task fails with:

```
Processing chart at "/tmp/build/put/git-source/charts/sample"...
22:42:14
Performing "helm package"...
22:42:14
Inspecting chart file: "/tmp/tmp-7Suy4nUQnZaLV/sample-v0.1.0.tgz"...
22:42:14
Uploading chart file: "/tmp/tmp-7Suy4nUQnZaLV/sample-v0.1.0.tgz"...
22:42:14
An error occured while uploading the chart: "404 - Not Found".
```

This is my resource:

```
  - name: chartmuseum
    type: chartmuseum
    source:
      chart_name: sample
      server_url: https://registry.elpenguino.net/chartrepo/
      basic_auth_username: admin
      basic_auth_password: mypassword
```

And this is my task:

```
      - put: chartmuseum
        params:
          chart: git-source/charts/sample
```

I've confirmed that the repo is working by manually pushing a chart, as follows:

```
# helm repo list
NAME                                    	URL
stable                                  	https://kubernetes-charts.storage.googleapis.com/
<snip>
retort                                  	https://registry.elpenguino.net/chartrepo/retort
# ± helm push sample retort
Pushing sample-v0.1.0.tgz to retort...
Done.
```

Any ideas appreciated ;)

Cheers!
D

**Comment by k-tipp** [comment]
I think it will work when you append the project to the server_url property:

> server_url: https://registry.elpenguino.net/chartrepo/retort
when you pushed your chart manually, you appended the project name too. I'm playing around with the same stuff at the moment.

**Comment by funkypenguin** [comment]
I've tried all combinations of:

```
      server_url: https://registry.elpenguino.net/chartrepo/
      server_url: https://registry.elpenguino.net/chartrepo/retort/
      server_url: https://registry.elpenguino.net/chartrepo/retort/sample/
```

And all fail at the same point. Any tips on how to debug further? :)

**Comment by funkypenguin** [comment]
I figured it out. See PR ^ above for all the changes necessary to make it fly. (Disclaimer: I'm not a Node developer!)

**Comment by k-tipp** [comment]
I'm glad you solved your problem. Actually I just start exploring Concourse and stumbled upon this resource.

**Comment by headcr4sh** [comment]
Closed by #9 
Thanks @funkypenguin ! :-)
